shared/libpng-util/Makefile

```diff
@@ -1,5 +1,13 @@
 .PHONY: libpng-util.a clean .FORCE
 
+# I tried include ../../Default.mk and it had a warning saying overriding .FORCE
+OS := $(shell uname -s)
```

Comment:
shouldnt need this here. You might need to modify the shell makefile to include config.mk tho